runtime/jcl/common/mgmtmemory.c

```diff
@@ -702,9 +703,72 @@ Java_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificati
 				return;
 			}
 
-			(*env)->CallVoidMethod(env, threadInstance, helperGCID,
-					gcName, gcAction, gcCause,
-					(jlong)gcInfo->index, (jlong)gcInfo->startTime, (jlong)gcInfo->endTime,
+			/* Error tracepoint for the cases
+			 * the value of init or max is negative but not -1; or
+			 * used is greater than the value of committed; or
+			 * committed is greater than the value of max if max is not -1.
+			 */
+			if (TrcEnabled_Trc_JCL_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificationLoop_memoryUsageError
+				|| TrcEnabled_Trc_JCL_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificationLoop_memoryUsage
+				) {
```

Comment:
This line should be indented the same as line 711. Similar comment for line 734.